tests/framework/cli/test_starters.py

```diff
@@ -314,17 +314,6 @@ def test_parse_add_ons_invalid_range(input, capsys):
     assert message in capsys.readouterr().err
 
 
-@pytest.mark.parametrize(
-    "input,first_invalid",
-    [("0,3,5", "0"), ("1,3,7", "7"), ("0-4", "0"), ("3-7", "7")],
-)
-def test_parse_add_ons_invalid_selection(input, first_invalid, capsys):
```

Comment:
This test was removed as this validation step is no longer performed by `_parse_add_ons_input`